### ISSUE TITLE
Node v10.18.0 (LTS)

### ISSUE THREAD

**Issue by outsideris** [nodejs/nodejs-ko#912]
https://nodejs.org/en/blog/release/v10.18.0/

```
npm run scaffold https://raw.githubusercontent.com/nodejs/nodejs.org/master/locale/en/blog/release/v10.18.0.md 2019-12-17
```

**Comment by MJayJang** [comment]
담당하겠습니다!

**Comment by yous** [comment]
Closed via #918.
